src/index.html

```diff
@@ -10,23 +10,36 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <main class="main">
+      <form class="search-bar">
+        <label for="id1"></label>
+        <input
+          id="id1"
+          type="text"
+          data-qa="big"
+          action="#"
+          placeholder="Try “Los Angeles“"
+          class="search-bar__big input"
+        />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+        <label for="id2"></label>
+
+        <input
+          id="id2"
+          type="text"
+          data-qa="small"
+          action="#"
+          placeholder="Try “Los Angeles“"
+          class="search-bar__small input"
+        />
```

Comment:
HTML Formatting: If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.